test/reporters/doc.spec.js

```diff
@@ -3,6 +3,8 @@
 var reporters = require('../../').reporters;
 var Doc = reporters.Doc;
 
+var runnerEvent = require('./helpers.js').runnerEvent;
```

Comment:
I'm not sure what to call this, but `runnerEvent` isn't descriptive enough.

Also, I'm not sure `runner` needs to be created as an empty object in `beforeEach`...

Instead, maybe this could just be a factory function for a `runner` object?  It would then create `.on` and `.once` as appropriate.

Then call it something like `createMockRunner`